aptos-move/e2e-tests/src/executor.rs

```diff
@@ -377,6 +381,66 @@ impl<O: OutputLogger> FakeExecutorImpl<O> {
         self
     }
 
+    /// Enable code coverage collection for this executor. This is thread safe,
+    /// but all threads should use the same `file_base_name` to avoid logical confusion. If
+    /// multiple threads with the same file generate tracing info concurrently, it will be
+    /// accumulated as expected. See `save_code_coverage` how to persist coverage info
+    /// and how to analyze it.
+    pub fn enable_code_coverage(&self, file_base_name: &str) {
```

Comment:
This only works with `nextest` because it runs test per process, otherwise setting this in some test enables this feature for all other concurrent tests or tests running after in the same process? We kind of have the same issue with paranoid mode, but we tests it in `third_party` via config.... Are we fine breaking `cargo test` for all e2e tests / making them slower?